client/components/marketing-survey/cancel-jetpack-form/index.tsx

```diff
@@ -286,17 +319,28 @@ const CancelJetpackForm: React.FC< Props > = ( {
 				{ disabled ? cancellingText : cancelText }
 			</Button>
 		);
-		const firstButtons: [ ButtonType ] = [ close ];
-
 		const loading = (
 			<Button disabled={ true } busy={ true } primary>
 				{ translate( 'Loading' ) }
 			</Button>
 		);
+		const backToSite = {
+			action: 'close',
+			isPrimary: true,
+			disabled: disabled,
+			label: translate( 'Back to my website' ),
```

Comment:
I find the label a bit misleading here. I would imagine I would be redirected to WPadmin instead of the purchases page.

Reply:
hmm, good point. Perhaps "back to my purchases" would be better.